Buildings/Resources/Scripts/Dymola/Controls/OBC/FDE/PackagedRTUs/Validation/BASController.mos

```diff
@@ -0,0 +1,86 @@
+// Script generated by Dymola Tue Aug 18 15:21:13 2020
+// Plot commands
```

Comment:
Please check how the `*.mos` scripts are usually structured in our sequence packages. We usually do not define any values in `mos` scripts.

Reply:
Per our review today, I'm going to rebuild all of the .mos files without relying on the automated script builder.